packages/controls/src/testing/test-definition.ts

```diff
@@ -2,10 +2,18 @@ import { ControlDataTypeKey } from '../common/types'
 import { type ValueType } from '../controls/associated-types'
 import { ControlDefinition } from '../controls/definition'
 
+function toV0<Def extends ControlDefinition>(definition: Def) {
+  const DefinitionClass = definition.constructor as any
+  return DefinitionClass.deserialize({
+    type: DefinitionClass.type,
+    config: definition.config,
+  })
+}
```

Comment:
Yep, it's dropping the version field. I didn't want to introduce a method to downgrade a def since the only place we are using this is tests.

Reply:
So you don't think there should be an explicit way to instantiate a particular definition at a particular version? Seems a bit roundabout to do it via deserialize

Reply:
We can devise something more explicit, but, to me, constructing a v0 definition via deserialization actually felt on kind of point since this is the only way we are going to get a v0 definition in the current version of the runtime.